src/chimera/__init__.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python 
+
+#==============================================================================
+
+__version__ = '0.1.0'
```

Comment:
maybe set in another file. `__about_.py` and include inside setup.py to use it.
`about = {}
version_file = pathlib.Path('.') / 'src' / 'chimera' / '__about__.py'
exec(version_file.read_text(), about)
`

and then about['version']

Reply:
Ok, updated. I believe that this way of setting the version is the solution that you suggested earlier for the phoenics repo. Which strategy should we pick going forward?